_config.yml

```
@@ -6,6 +6,7 @@ title: Repair Café Malmö
 navigation:
   - {name: 'Home', link: '/'}
   - {name: 'About Us', link: '/about'}
+  - {name: 'Cheat Sheet', link: '/cheatsheet'}
```

```suggestion
  - {name: 'How To Repair', link: '/howtorepair'}
```
Actually I changed the name here as well